### ISSUE TITLE
Sidewalk Gallery: New UI: Doesn't look good at certain window sizes

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2590]
## With expanded card closed
![image](https://user-images.githubusercontent.com/1621749/122487735-cb6b9e00-cf90-11eb-8f52-0c56fa54663e.png)

## With expanded card open
![image](https://user-images.githubusercontent.com/1621749/122487724-c4449000-cf90-11eb-8604-1cf66eaddcf7.png)

![image](https://user-images.githubusercontent.com/1621749/122487706-b989fb00-cf90-11eb-82e6-aed4d919c0ad.png)

One simple fix but won't fix everything is to hide the severity and tags divs when screen size gets too narrow.
![image](https://user-images.githubusercontent.com/1621749/122487840-0372e100-cf91-11eb-8a1c-bce8cda2f842.png)

**Comment by jonfroehlich** [comment]
Just marking that this is still an issue after @ThatOneGoat's new (amazing) UI update:

![image](https://user-images.githubusercontent.com/1621749/123325558-03bd3000-d4ed-11eb-8ee0-edcb4f69a7ba.png)

**Comment by ThatOneGoat** [comment]
Yup, still something on my TODO list. I've been pushing it back though because I still think there are some more higher-priority bugs to amend.

**Comment by jonfroehlich** [comment]
Just made a quick video to help highlight many of the issues when resizing your window: https://youtu.be/iXau1o-MYF4

I think one helpful fix would be to have a min-width for the full canvas space for which we no longer attempt to reduce the size of things or rearrange them. Secondly, I think we need a max/min height for the expanded card.

**Comment by jonfroehlich** [comment]
I have fixed this for most window sizes but just discovered that I did miss some.

## Working
<img width="2437" height="1706" alt="Image" src="https://github.com/user-attachments/assets/7d2478ad-bdec-4390-b21b-68559b75f360" />

<img width="1243" height="1697" alt="Image" src="https://github.com/user-attachments/assets/21ea121e-f7da-46cd-b771-8b92e16f097e" />

<img width="1266" height="1738" alt="Image" src="https://github.com/user-attachments/assets/aa994aeb-e3c4-4a4f-b521-b7847bf35121" />

## Not working
This is mostly because we haven't written Gallery to be mobile responsive yet. 

<img width="743" height="1735" alt="Image" src="https://github.com/user-attachments/assets/684ca410-4350-4fd2-8a63-004b8696ef6f" />

<img width="764" height="1384" alt="Image" src="https://github.com/user-attachments/assets/e3b1e9bf-cf04-4a0d-9bfa-b89a6f3bd091" />

**Comment by misaugstad** [comment]
@jonfroehlich Has reworked Gallery to work much better at smaller window sizes!
